src/finalFrame.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+class FinalFrame {
```

Comment:
It seems like a `FinalFrame` is mostly the same as a `Frame`. Is there a way to remove this class and instead have the `Frame` class capture the necessary functionality?